docs/sdks/sdk-ref-python.md

```diff
@@ -204,6 +230,38 @@ The SDK hashes input values before sending them. This ensures that raw email add
         reason = unmapped_identity.get_reason()
    ```
 
+### Monitor Rotated Salt Buckets
+
+To monitor salt buckets, follow these steps.
+
+1. Create an instance of `IdentityMapClient` as an instance variable or reuse the one from [Map Personal Data to Raw EUIDs:](#map-personal-data-to-raw-euids)
+
+   ```py
+   client = IdentityMapClient(base_url, api_key, client_secret)
+   ```
+
+2. Call a function that takes the timestamp string as input and generates an `IdentityBucketsResponse` object. The timestamp string should be in ISO 8601 format: `YYYY-MM-DD[*HH[:MM[:SS[.fff[fff]]]][+HH:MM[:SS[.ffffff]]]]`.
+The following examples are valid timestamp strings:
+   - Date in local timezone: `2024-08-18`
+   - Date and time in UTC: `2024-08-18T14:30:15.123456+00:00`
+   - Date and time in EST: `2024-08-18T14:30:15.123456-05:00`
```

Comment:
EST is probably irrelevant here. Let's try Central European Summer Time 
```
 - Date and time in CEST: `2024-08-18T14:30:15.123456+02:00`
```

Reply:
Thank you! Totally. Done.